server.js

```diff
@@ -2,12 +2,16 @@
 const express = require('express');
 const cors = require('cors');
 const superagent = require('superagent');
+const pg = require('pg');
 require('dotenv').config();
 
 
 //====================== app ===================
 const app = express();
 app.use(cors());
+const DATABASE_URL = process.env.DATABASE_URL;
+const client = new pg.Client(DATABASE_URL);
+client.on('error', error => console.log(error));
```

Comment:
Nice I haven't seen that before but I like it. does it otherwise not log the error?